src/css/header.css

```diff
@@ -0,0 +1,39 @@
+#header {
+  display: -webkit-box;
+  display: flex;
+  height: 5em;
+  max-height: 5em;
+  -ms-flex-line-pack: center;
+  align-content: center;
+  padding: 2em 2em 6em 6em;
+}
+
+#header .logo {
+  height: 100%;
+  margin-right: 5em;
+  border: 2px solid black;
+  border-radius: 50%;
+  overflow: hidden;
+}
```

Comment:
* I also suggest you use a non-responsive unit like 'px' or 'em' for logo width and then set its height to auto.  This will prevent it from reducing in size to totally disappearing, while we move to smaller screens, making the whole interface clean and consistent.